shared/src/store/action-reducers/utils/calculate-treatments.ts

```diff
@@ -30,7 +30,11 @@ function caterFor(
     catersFor: Mutable<CatersFor>,
     patient: Mutable<Patient>
 ) {
-    const status = patient.visibleStatus ?? 'yellow'; // Treat not pretriaged patients as yellow.
+    let status = patient.visibleStatus;
+    // Treat not pretriaged patients as yellow.
+    if (status === 'white') {
+        status = 'yellow';
+    }
```

Comment:
```suggestion
    // Treat not pretriaged patients as yellow.
    const status = patient.visibleStatus === 'white'? 'yellow' : patient.visibleStatus;
```

`let` makes the code more complicated to read because `status` could potentially be reassigned anywhere in the next 50 lines...